Content.Shared/Dice/DiceComponent.cs

```diff
@@ -34,4 +34,6 @@ public sealed partial class DiceComponent : Component
     [AutoNetworkedField]
     public int CurrentValue { get; set; } = 20;
 
+    [DataField("diceBomb")]
```

Comment:
Also please, separate component

Dice roll logic and explosive dice logic should not be mixed in the same component and system, as they do completely different things.